src/openklant/components/klanten/models/klanten.py

```diff
@@ -6,12 +6,21 @@
 from django.db import models
 from django.utils.translation import gettext_lazy as _
 
+from vng_api_common.exceptions import Conflict
 from vng_api_common.fields import BSNField, RSINField
 from vng_api_common.models import APIMixin
 
 from .constants import GeslachtsAanduiding, KlantType, SoortRechtsvorm
 
 
+class KlantManager(models.Manager):
+    def get_next_klantnummer(self):
+        id_max = Klant.objects.all().aggregate(models.Max("klantnummer"))[
+            "klantnummer__max"
+        ]
+        return int(id_max) + 1 if id_max else 1
```

Comment:
Since ` klantnummer`  is a char field and it's writable in the API, it's possible that it can be filled with the value which can't be converted into `int` (for example, 'KLANT11')
So if you have such value in DB the code would crush with `ValueError`